samples/csharp_dotnetcore/14.nlp-with-dispatch/NLP-With-Dispatch-Bot.csproj

```diff
@@ -7,8 +7,8 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="3.1.1" />
-    <PackageReference Include="Microsoft.Bot.Builder.AI.Luis" Version="4.8.0" />
-    <PackageReference Include="Microsoft.Bot.Builder.AI.QnA" Version="4.8.0" />
+    <PackageReference Include="Microsoft.Bot.Builder.AI.Luis" Version="4.9.0-preview-200413-119524" />
```

Comment:
can we check-in samples with daily builds?, I think no, @vishwacsena  please confirm?

Reply:
I think @vishwacsena  already confirmed to us on email

>> Thanks for the update. We always need to have samples use published packages to npm/ myget and not the nightly feed. Given these samples require latest packages, the right solution would be to get these in PR once we have 4.9 packages published to npm/ myget. At the moment we are targeting mid-may (our R9 release) for the release. 

so, you can use this nightly build for proper end2end testing. But, you should be good check-in just the host name change alone, as that don't break with prod sdk. That will help us make progress with nightly builds on templates branch as well.